lib/field.ml

```diff
@@ -9,6 +9,7 @@ type value =
   | `String of string
   | `Bytes of bytes
   | `Time of Time.t
+  | `Json of string
```

Comment:
Would it not be better to use the `` `String`` case for JSON, as well?  I didn't write the original code, but the intention seem to me that the `value` type only contains the cases which have different representation on the OCaml side, e.g. omitting the various integer variants.